tests/src/test/java/alluxio/testutils/PersistenceTestUtils.java

```diff
@@ -92,9 +93,13 @@ public static void pauseScheduler(LocalAlluxioClusterResource resource) {
    */
   public static void resumeScheduler(LocalAlluxioClusterResource resource) {
     FileSystemMaster nestedMaster = getFileSystemMaster(resource);
-    BlackHoleMap<Long, ExponentialTimer> persistRequests =
-        Whitebox.getInternalState(nestedMaster, "mPersistRequests");
-    Whitebox.setInternalState(nestedMaster, "mPersistRequests", persistRequests.getInnerMap());
+    try {
+      BlackHoleMap<Long, ExponentialTimer> persistRequests =
+          Whitebox.getInternalState(nestedMaster, "mPersistRequests");
+      Whitebox.setInternalState(nestedMaster, "mPersistRequests", persistRequests.getInnerMap());
+    } catch (ClassCastException e) {
+      Whitebox.setInternalState(nestedMaster, "mPersistRequests", new ConcurrentHashMap<>());
```

Comment:
I'm not sure what is going on here. Can you add a comment?

Reply:
updated